language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/TypeDescContextTest.java

```diff
@@ -51,6 +51,7 @@ public String getTestResourceDir() {
 
     @Override
     public List<String> skipList() {
-        return Arrays.asList();
+        // Skip this test due to issue: https://github.com/ballerina-platform/ballerina-lang/issues/37039
```

Comment:
This issue is closed now. Can you check and enable this separately may be?

Reply:
#37039 is resolved and add relevant stages in `2.0-stage` branch.
So eventually it will merge into `2.x` branch and it will merge into the `master` branch as well.